src/LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License (MIT)
+Copyright (c) <year> <copyright holders>
```

Comment:
@aanarres Oh, I can't really comment - I'm not a member of the platform; I'm just a bit nosey :wink: